src/basic/parse-util.h

```diff
@@ -11,6 +11,28 @@
 
 typedef unsigned long loadavg_t;
 
+#define TPM2_PCRS_MAX 24U
+
+typedef enum PcrIndex {
+        PCR_CORE_FIRMWARE      = 0,
+        PCR_PLATFORM_CONFIG    = 1,
+        PCR_EXTERNAL_CODE      = 2,
+        PCR_EXTERNAL_FIRMWARE  = 3,
+        PCR_BOOT_LOADER        = 4,
+        PCR_BOOT_LOADER_CONFIG = 5,
+        PCR_HOST_PLATFORM      = 6,
+        PCR_SECURE_BOOT_POLICY = 7,
+        PCR_KERNEL_INITRD      = 9,
+        PCR_IMA                = 10,
+        PCR_KERNEL_IMAGE_PHASE = 11,
+        PCR_KERNEL_CONFIG      = 12,
+        PCR_SYSEXT_IMAGE       = 13,
+        PCR_SHIM_POLICY        = 14,
+        PCR_SYSTEM_IDENTITY    = 15,
+        _PCR_MAX               = TPM2_PCRS_MAX,
```

Comment:
Please call this `_PCR_INDEX_MAX_DEFINED` and `_PCR_INDEX_INVALID`.

Reply:
> Please call this `_PCR_INDEX_MAX_DEFINED` and `_PCR_INDEX_INVALID`.

Certainly.